Spots final

alexanderbrosovich opened the pull request:
prep for review

alexanderbrosovich committed:
Api first steps and refactoring

alexanderbrosovich committed:
.cards fill -> fit

alexanderbrosovich committed:
refactoring, incorperating avatar button

alexanderbrosovich committed:
css for delete modal

alexanderbrosovich committed:
api newPost

alexanderbrosovich committed:
api modal delete

alexanderbrosovich committed:
error in opening deleteModal

alexanderbrosovich committed:
dubugging

alexanderbrosovich committed:
debugging

alexanderbrosovich committed:
debugging

alexanderbrosovich committed:
debugging

alexanderbrosovich committed:
debugging

alexanderbrosovich committed:
debbuging

alexanderbrosovich committed:
debugging

alexanderbrosovich committed:
debugging

olaracode reviewed (changes requested):
Hi Alex! Really good job while working on this sprint!

Almost everything works as expected, and what doesn't work as expected are an easy fix!

The main Issue I'm seeing is that after I update the avatar image the change doesn't get reflected right away and when I refresh the website it doesn't change either. The request is working it's just that you're not using the value correctly(More information on the inline comments)

All the other comments are rather small!

The best of luck while working on this correction

alexanderbrosovich committed:
moved modal under page class
edit-avatar-modal
modal-delete

 set the avatar image src
 editAvatarInfo value -> data.avatar

 adjusted profile__avatar-button icon

 setButtonText argument true -> false for api
    .editAvatarInfo

olaracode reviewed (commented):
Hi Alex!

Thanks for taking care of the comments I made, overall great job! The issue with the profile persisted, but I realize that my previous comment was kind of vague, so I left a better worded comment on line 74 with the solution to the main issue,

Please address this issue as it's a rather simple fix but it changes the outcome significantly, 

The best of luck while working on the next stages of the bootcamp!

alexanderbrosovich committed:
set const for profileAvatar
